transcom/mymove
# B-20235-INT Make feedback review button available when docs kicked back

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20235)

## Summary

It was requested that until the work is complete to send all completed ppms to "CLOSEOUT_COMPLETE" status, we display the feedback if any items were rejected, regardless of document review status.

### How to test

1. Get a ppm to document review stage
2. Login as an SC and reject a document
3. Login as customer and confirm you can see the View Feedback button
4. Try and break
